src/schemas/components/SchemaFormProperty.vue

```diff
@@ -114,7 +116,7 @@
     },
   })
 
-  if (!props.skipDefaultValueInitialization && !isDefined(props.value) && isDefined(property.value.default)) {
+  if (!skipDefaultValueInitializationForAllProperties && !props.skipDefaultValueInitialization && !isDefined(props.value) && isDefined(property.value.default)) {
```

Comment:
This is the primary change. Checking both the local prop and the global form setting to see if the default value initialization should be skipped.